Snakefile

```diff
@@ -219,6 +219,19 @@ rule cluster_network:
     # group: 'build_pypsa_networks'
     script: "scripts/cluster_network.py"
 
+
+rule add_extra_stores:
+    input:
+        network='networks/{network}_s{simpl}_{clusters}.nc',
+        tech_costs=COSTS,
+    output: 'networks/{network}_s{simpl}_{clusters}.nc'
```

Comment:
Yes, I think it is necessary to reflect this rule in the output file names. I like `_ec` (for extra components) or `C` (for Components). I would avoid `_ac` (for additional components) due to ambiguity with AC power flow.
```suggestion
    output: 'networks/{network}_s{simpl}_{clusters}_ec.nc'
```